src/Calendar.js

```diff
@@ -13,6 +13,18 @@ import { DateTime } from 'luxon'
 
 import { getStartOfSession } from './Toolbar'
 
+// Monkey patch rrulePlugin for FullCalendar to fix https://github.com/fullcalendar/fullcalendar/issues/5273
+// (Recurring events don't respect timezones in FullCalendar)
+// We simply replace the expand function here: https://github.com/fullcalendar/fullcalendar/blob/ede23c4b2bf0ee0bb2cbe4694b3e899a09d14da6/packages/rrule/src/main.ts#L36-L56
+// With a custom version below
+rrulePlugin.recurringTypes[0].expand = function (errd, fr, de) {
+  return errd.rruleSet.between(
+    fr.start,
+    fr.end,
+    true, // inclusive (will give extra events at start, see https://github.com/jakubroztocil/rrule/issues/84)
+  ).map(date => new Date(de.createMarker(date).getTime() + date.getTimezoneOffset() * 60 * 1000))
```

Comment:
maybe return a marker just to be safe?
```suggestion
  ).map(date => de.createMarker(new Date(date.getTime() + date.getTimezoneOffset() * 60 * 1000))))
```

Reply:
`DateMarker` is a type alias for `Date`, so the difference here is whether we're using createMarker before or after applying the timezone offset.

I think we should use `createMarker` before applying an offset in case the offset would make it misbehave, so I'll leave it as is. I can't tell any difference with this change however

Reply:
ah ok. I was thinking future compatibility, but that's very fair too